public/pages/techtalk.html

```diff
@@ -97,20 +103,21 @@ <h4>Past Talks</h4>
     <h2>Featured Tech Talk Speaker</h2>
 
     <div class="imgcontain">
-      <img src="../img/tech-talk_paul-beresuita.png">
+      <img src="../img/tech-talk_Erin-Shaw.png">
     </div>
 
     <div id="zoom" class="fragment-destination"></div><style>.fragment-destination {position: relative;top: -55px;}</style>
-    <h2>Watch the Tech Talk</h2>
-    <div class="boxed-in">
-      <iframe width="560" height="315" src="https://www.youtube.com/embed/nZ6l9r1atF8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-    </div>
+
   </main>
 
     <!-- footer 2.2.2 -->
     <div class="footer">
       <footer class="footer-container">
         <div class="footer-main-1">
+          <div class="boxed-in">
+      <p>The Zoom meeting will be available to join at <b>6:30pm EST</b> on Wednesday February 9th.</p>
+      <a class="button">Join Zoom Meeting Now</a>
```

Comment:
I would change the button to say something like "Zoom meeting will open at 6:00pm"